### ISSUE TITLE
[BUG] Auto-complete does not work for JSON files!

### ISSUE THREAD

**Issue by SonicandTailsCD** [Blockception/VSCode-Bedrock-Development-Extension#708]
**Describe the bug**
It's simple, actually. While typing, for example, `"minecraft:`, it does not show the possible properties. Heck, it won't even show an error after saving, for example, an invalid property: `"minecraft:behavior.fart": {}`. Obviously that's not valid but unfortunately it won't warn you it's invalid.

**To Reproduce**
1. Just make a new file and start writing. Some things will have auto-complete, some won't.

**Expected behavior**
Auto-complete to actually work and for invalid properties to be flagged, regardless of whether you have to save first.

**Screenshots**
![Screenshot proving this](https://github.com/user-attachments/assets/927ee5d6-d449-4080-995e-6e6a408a4766)
This isn't supposed to not be flagged.

**Desktop (please complete the following information):**
 - OS: Ubuntu 20.04.6 LTS
- VS Code version: 1.90.0 (can't upgrade)
 - Extension version: latest (v8.0.18)

**Comment by DaanV2** [comment]
Is this custom entity properties? or an component?

**Comment by Xterionix** [comment]
Full file and folder setup would be nice to know

**Comment by SonicandTailsCD** [comment]
> Is this custom entity properties? or an component?

I don't quite get what you mean, but if I were to guess, you're asking me if I'm just testing one component, or an entire custom entity.
The answer to this: "Custom entity". :)

**Comment by SonicandTailsCD** [comment]
> Full file and folder setup would be nice to know

I have VS Code open with both the resource and behavior packs loaded.

**Comment by Xterionix** [comment]
> I have VS Code open with both the resource and behavior packs loaded.

Yeah, that's not answering what I asked. There are many ways people load both the bp and rp.

**Comment by SonicandTailsCD** [comment]
> > I have VS Code open with both the resource and behavior packs loaded.
> 
> Yeah, that's not answering what I asked. There are many ways people load both the bp and rp.

Sorry. I meant, I have both folders loaded in my editor, as in, 2 folders, BP and RP, added to the workspace.

**Comment by Xterionix** [comment]
Can you share the full file?

**Comment by SonicandTailsCD** [comment]
Well, it does this on any file, but sure. Here you go:
[bowser.json](https://github.com/user-attachments/files/18248001/bowser.json)
